### PR TITLE
{vis}[foss/2015a] SPIDER 22.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/94db658a55283e6a25ce for a full test report.

File: easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ an image processing system for electron microscopy."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'static': True}
+
+source_urls = ['http://spider.wadsworth.org/spider_doc/spider/download/']
+sources = ['spiderweb.%(version)s.tar.gz']
+
+patches = [
+    'SPIDER-%(version)s_fix-syntax-ifort.patch',
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-foss-2015a.eb]: do you need the ifort patch for foss?

Comment (boegel) [inline easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-foss-2015a.eb]: probably, yes, it's actual (syntax) errors in the code

I reported these upstream btw.

**Comment by boegel** [comment]
going in, thanks for the review @JensTimmerman !

**Pull request merged**